src/sherpa_ai/task_agent.py

```diff
@@ -152,13 +154,19 @@ def run(self, task: str) -> str:
                 logger_step["reply"] = reply_json
             except json.JSONDecodeError:
                 logger_step["reply"] = assistant_reply  # last reply is a string
+            if self.agent_config.verbose:
+                self.verbose_logger.log(f"```{assistant_reply}```")
+
             self.logger.append(logger_step)
 
-            ########## Serial Verbose Feature #######
-            try:
-                formatted_logger_step = show_commands_only(logger_step)
-            except Exception as e:
-                logger.error(e)
+            # Serial Verbose Feature
```

Comment:
conditionals like this are a little easier to read if you structure them with the positive case first, e.g.

```
if self.agent_config.verbose
    formatted_logger_step = logger_step
else
    try ...
```